packages/twenty-front/src/modules/activities/emails/right-drawer/components/RightDrawerEmailThread.tsx

```diff
@@ -43,11 +43,14 @@ export const RightDrawerEmailThread = () => {
 
   useRegisterClickOutsideListenerCallback({
     callbackId:
-      'EmailThreadClickOutsideCallBack-' + (thread.id ?? 'no-thread-id'),
+      'EmailThreadClickOutsideCallBack-' + (thread?.id ?? 'no-thread-id'),
```

Comment:
I'm a bit worried about these thread?.id, the RightDrawerEmailThread content should not be loaded at all in this case. We can't add a if in this component as hooks cannot be called conditionnaly but we can do a parent that will load to children and block with an error if thread is not defined / null.

Reply:
Right, thx. I'm actually not 100% sure this code is still relevant.